instrumentation/netty/netty-4.0/src/main/java/io/opentelemetry/javaagent/instrumentation/hypertrace/netty/v4_0/client/HttpClientResponseTracingHandler.java

```diff
@@ -104,7 +104,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
       span.setAttribute(SemanticAttributes.HTTP_STATUS_CODE, httpResponse.getStatus().code());
       span.setStatus(HttpStatusConverter.statusFromHttpStatus(httpResponse.getStatus().code()));
     }
-    if (msg instanceof FullHttpMessage || msg instanceof LastHttpContent) {
```

Comment:
Do we not want to check the FullHttpMessage ever now?

Reply:
In this case, `FullHttpMessage` extends `LastHttpContent` so this check was redundant